Packs/PAN-OS/Integrations/Panorama/Panorama.yml

```diff
@@ -173,6 +173,31 @@ script:
   - arguments:
     - description: Commit description.
       name: description
+    - description: To commit admin-level changes on a firewall, include the administrator
+        name in the request.
+      name: admin_name
+    - auto: PREDEFINED
+      description: (True/False) Force Commit
+      name: force_commit
+      defaultValue: 'false'
+      predefined:
+        - 'true'
+        - 'false'
+    - auto: PREDEFINED
+      description: (True/False) Partial commit while excluding shared objects and device and network configuration.
+      name: exclude_device_network_configuration
+      defaultValue: 'false'
+      predefined:
+        - 'true'
+        - 'false'
+    - auto: PREDEFINED
+      description: (True/False) Partial commit while excluding shared objects and
```

Comment:
The description here should be the same as `exclude_device_network_configuration`?

Reply:
good catch, thanks